### DISCUSSION TITLE
urllib3.exceptions.MaxRetryError happened

### DISCUSSION THREAD

**Discussion by kelvinwang139** [PyTorchLightning/lightning-flash#1309] [Q&A] [answered]
Hi，

During I run flash command, the following error occured, could you please help on it?
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /airctic/storage/master/SpaceGrotesk-Medium.ttf (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002494CA186D0>: Failed to establish a new connection: [Errno 11004] getaddrinfo failed'))

**Comment by ethanwharris** [comment, marked as answer]
Hi @kelvinwang139, it looks like there is an issue in IceVision getting the font file from Github. You should be able to fix it by manually downloading this file: https://raw.githubusercontent.com/airctic/storage/master/SpaceGrotesk-Medium.ttf
and placing it in `~/.icevision/fonts/`

Hope that helps :smiley:

**Reply by kelvinwang139** [reply]
Thanks for you clear answer and it works.
